### DISCUSSION TITLE
Multiple custom alerts?

### DISCUSSION THREAD

**Discussion by 0x2142** [TwiN/gatus#477] [Q&A] [answered]
Hello - 
   Wondering if there is a method to configure multiple custom alert methods? I'm not seeing any way to do this immediately, but want to make sure I'm not missing anything. I see the ability to provide group overrides for other alerting methods, but not for custom alerts.

I use Gotify for most of my alerting & would like to create multiple profiles with different alert priorities. Also, I do have at least one service monitor where I would want to trigger a webhook to restart a component if there is a failure. 

Thanks in advance!

**Comment by TwiN** [comment, marked as answer]
There has been a request to support Gotify in the past #239, perhaps we should implement it once and for all instead of relying on the custom alerting provider.

But to answer your question, you're right, you cannot configure multiple custom alerting providers, only one, and rather than allowing multiple custom providers, I do think it would be a better idea to support gotify under its own provider instead.

**Reply by 0x2142** [reply]
Okay, no worries. Appreciate the response!
